ratbagd/ratbagd-device.c

```diff
@@ -274,7 +275,7 @@ ratbagd_device_get_capabilities(sd_bus *bus,
 
 const sd_bus_vtable ratbagd_device_vtable[] = {
 	SD_BUS_VTABLE_START(0),
-	SD_BUS_PROPERTY("Id", "s", NULL, offsetof(struct ratbagd_device, sysname), SD_BUS_VTABLE_PROPERTY_CONST),
+	SD_BUS_PROPERTY("Id", "s", NULL, offsetof(struct ratbagd_device, id), SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
We should document this in the DBus API.

Reply:
already there, see `doc/dbus.rst`

```
          A unique ID describing this device. This ID should not be used for presentation to
          the user. This ID may be recycled when the device is removed. The
          content of the ID is undefined, the client should treat it as an
          opaque string.
```

Reply:
Oops, missed that. Sorry.